ui/src/components/ProfileCard/styles.module.scss

```diff
@@ -42,7 +42,8 @@
   right: 10vh;
 }
 
-@media only screen and (max-width: 700px) {
+@import '../../index';
```

Comment:
Let's create an issue for deprecating relative imports.

from `../../index' to `/index` or `@index`

Reply:
Done.